csrc/scheduler/mma_utils.cpp

```diff
@@ -1210,6 +1382,24 @@ std::vector<MatmulDimRole> canonicalizeMmaTvOrdering(
   return roles;
 }
 
+void mergeCanonicalAbstractTensor(AbstractMatmulTensor& abstract_tensor) {
```

Comment:
This is currently unused but will be useful for doing `AbstractTensor` scheduling in the future.